apps/src/aiDifferentiation/AiDiffContainer.tsx

```diff
@@ -58,6 +64,33 @@ const AiDiffContainer: React.FC<AiDiffContainerProps> = ({
         className={style.aiDiffContainer}
         style={open ? undefined : {display: 'none'}}
       >
+        <div className={classNames(style.aiDiffHeader, 'ai_diff_handle')}>
```

Comment:
This was missed in my original split PR, but should have been in the container and not the chat.